azkaban-common/src/main/java/azkaban/executor/JdbcExecutorLoader.java

```diff
@@ -1364,6 +1382,52 @@ public String handle(ResultSet rs) throws SQLException {
     }
   }
 
+  private static class FetchRecentlyFinishedFlows implements
+    ResultSetHandler<List<ExecutableFlow>> {
+    // Execution_flows table is already indexed by end_time
```

Comment:
A bit confused what this comment want to tell readers? Should it be "Execution_flows table should index end_time"